Sources/MapboxDirections/RouteLeg.swift

```diff
@@ -274,7 +276,7 @@ open class RouteLeg: Codable {
      */
     open var expectedTravelTime: TimeInterval
 
-    open var administrationRegions: [AdministrationRegion]?
+    open var administrativeRegions: [AdministrativeRegion]?
```

Comment:
If we keep this property and `RouteStep.administrativeRegionIndicesByIntersection` around, then we need to document this property and also address #491.

Reply:
Done